lib/ddr/events/virus_check_event.rb

```diff
@@ -13,18 +13,6 @@ def to_solr
           Ddr::IndexFields::LAST_VIRUS_CHECK_OUTCOME => outcome }
       end
 
-      # Message sent by ActiveSupport::Notifications
-      def self.call(*args)
```

Comment:
The override of `Ddr::Events::Event.call` is no longer needed b/c not using ddr-antivirus.